NEWS.md

```diff
@@ -0,0 +1,21 @@
+# dccvalidator (development version)
+
+- (develop branch) Add `check_complete_ids()`, which includes new config option designating the table to compare against for existing specimen and individual IDs
```

Comment:
I would recommend taking this section out, rebasing develop onto master (so the news file gets added to develop) and then adding this news bullet only on develop.

Reply:
That is a good idea for just having that news bullet on develop.

That whole news section for the development version was intended for incremental additions on master between the latest release and the present moment (idea taken from Jim Hester's lintr package NEWS.md). The PR template update included in this PR has a request to add info there if there is a change to functionality. I would still like to keep that ability so we can track the most relevant changes in a more visible way. Would it be better to remove that whole section for this specific PR, do all the CRAN/release stuff, then do another PR just adding that top header again for future updating?

Reply:
Yes, I think that would be ideal 👍